src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,80 @@
+package duke;
+
+import java.util.ArrayList;
+
+//contains the task list e.g., it has operations to add/delete tasks in the list
+public class TaskList {
+
+    private ArrayList<Task> log;
+
+    TaskList() {
+        log = new ArrayList<>();
+    }
+
+    public void setLog(ArrayList<Task> log) {
+        this.log = log;
+    }
+
+    public ArrayList<Task> getLog() {
+        return log;
+    }
+
+    public Task addTodo(String s) throws DukeTaskException {
```

Comment:
Is it possible to generalise addTodo etc. to maybe something like addTask(Task newTask)?